site/gdocs/components/ArticleBlocks.tsx

```diff
@@ -29,13 +30,17 @@ export const ArticleBlocks = ({
     <>
         {blocks.map((block: OwidEnrichedGdocBlock, i: number) => {
             return (
-                <ArticleBlock
+                <BlockErrorBoundary
```

Comment:
moving the error boundary outside of articleblock means that we don't get the benefit of the error boundary when articleblock is used on its own (and not through articleblocks). 
For instance, an error thrown by [a social block in author pages](https://github.com/owid/owid-grapher/blob/ce36882c8ac95d76fbad5d518cad8075ec855bea/site/gdocs/pages/Author.tsx#L52) wouldn't be caught.

Reply:
I explained [here](https://github.com/owid/owid-grapher/pull/4614#discussion_r1975793288) why I moved it outside. The error should still be captured in Sentry, but it's true that a bigger part of the page would break, potentially the whole page.

To make it work properly for all `ArticleBlock`s, we'd need to introduce another layer of indirection. I guess it might be worth it, so I've added that.